Dockerfile

```diff
@@ -1,25 +1,20 @@
-FROM harbor.k8s.libraries.psu.edu/library/ruby-3.1.2-node-16:20240701 as base
+FROM harbor.k8s.libraries.psu.edu/library/ruby-3.1.2-node-16:20240701 AS base
 # Isilon has issues with uid 2000 for some reason
-# change the app to run as 201
-ARG UID=201
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+# change the app to run as 201, or as your local user id (1000 - jrp22)
```

Comment:
Should this say 3000?

Reply:
I was having some local issues and those are resolved.  I set this back to what it was prior (201)